pull_request_template.md

```diff
@@ -27,7 +36,10 @@ If relevant, include a screenshot or screen capture of the changes.
 <summary>After:</summary>
 
 </details>
+-->
 
+<!--
 ## 🚀 Notes for Deployment
 
 Include any special instructions for deployment.
+-->
```

Comment:
I believe there'd been [an action item from a previous engineering huddle](https://docs.google.com/document/d/1jgcAO32MS5KsVOx6grg2W0sU13j_cP0L0g8xxEq6WU0/edit#heading=h.unzn9c2ktfe8) to remove this section altogether. Maybe we can do that here?

```suggestion
```

Reply:
Good call! Done